utils/installer.py

```diff
@@ -93,6 +93,10 @@ def install_satellite(args):
     update_section('satellite', 'ssh_password', password)
     update_section('satellite', 'admin_username', admin_username)
     update_section('satellite', 'admin_password', admin_username)
+    # disable the version-locking for now
```

Comment:
Could you use `foreman-maintain packages unlock`  to instead, because satellite is already installed, we should not run `satellite-installer` again.
The second is could you add this change from LINE89?

Reply:
ok, will change it with next commit